swift_code_metrics/_graphics.py

```diff
@@ -16,18 +17,10 @@ def __init__(self, path=None):
     def bar_plot(self, title, data):
         plt.title(title)
         plt.ylabel(title)
-        bar_width = 0.75
-        opacity = 0.4
-        plotted_data = plt.barh(data[1], data[0], bar_width, alpha=opacity)
-
-        texts = []
-        for i, v in enumerate(data[0]):
-            texts.append(plt.text(v, i, f' {str(v)}', va='center', color='blue', size='smaller'))
-
-        adjust_text(texts, autoalign='x', only_move={'text': 'x'})
-
-        plt.legend(plotted_data, data[2], loc='lower right')
-
+        opacity = 0.8
+        plotted_data = plt.barh(data[1], data[0], color='blue', alpha=opacity)
```

Comment:
Are you removing the legend altogether?

Reply:
Yes, because a big legend is a bit confusing to visually map each framework with its short name. Instead is easier to add the full name with a slight rotation, next to bar. This is specially cleaner with code bases with many frameworks. What do you think about it?

Reply:
The idea is really nice. Just two notes:
- you don't need the `plotted_data` variable, so you can use the implicit assignment `_`
- I tried locally and I found out that sometimes the titles on the Y axis are truncated. Can you please double check that?

Reply:
Thanks for checking it! 
- They are truncated in case the name is too long? 
- Would it be a better solution to make the font smaller or to add the legend only in case the names are too long?